From 40ddc4c4e9eaf0d1ebb6ea3b44a4215d49855035 Mon Sep 17 00:00:00 2001
From: Arne Ruhnau <arne.ruhnau@target-sg.com>
Date: Wed, 16 Jul 2014 23:14:00 +0200
Subject: [PATCH 10/14] Added origin of the sdma firmware to WHENCE

---
 firmware/WHENCE | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/firmware/WHENCE b/firmware/WHENCE
index 0c4d96d..c511c96 100644
--- a/firmware/WHENCE
+++ b/firmware/WHENCE
@@ -852,3 +852,10 @@ Licence:
 Found in hex form in kernel source.
 
 --------------------------------------------------------------------------
+
+Driver: imx-sdma - SDMA scripts
+
+File: imx/sdma/sdma-imx6q.bin.ihex
+Version: 1.1
+
+Originates from git://git.freescale.com/imx/linux-2.6-imx.git
-- 
1.9.1

