commit 3a2cf3c4c6a6eadf1719b88b26c223f25979aac6
Author: Arne Ruhnau <arne.ruhnau@target-sg.com>
Date:   Sun Jul 13 03:11:13 2014 +0200

    Add GPU to the device tree

diff --git a/arch/arm/boot/dts/imx6dl-trizeps7-pconxs.dts b/arch/arm/boot/dts/imx6dl-trizeps7-pconxs.dts
index 7b682db..a789ee4 100644
--- a/arch/arm/boot/dts/imx6dl-trizeps7-pconxs.dts
+++ b/arch/arm/boot/dts/imx6dl-trizeps7-pconxs.dts
@@ -50,6 +50,10 @@
 
 };
 
+&gpu {
+	status = "okay";
+};
+
 &iomuxc {
 	trizeps7-pconxs {
 		pinctrl_i2c2_2: i2c2grp-2 {
diff --git a/arch/arm/boot/dts/imx6dl.dtsi b/arch/arm/boot/dts/imx6dl.dtsi
index 0a9c49d..035d47e 100644
--- a/arch/arm/boot/dts/imx6dl.dtsi
+++ b/arch/arm/boot/dts/imx6dl.dtsi
@@ -53,6 +53,18 @@
 	};
 
 	soc {
+		gpu: gpu {
+			compatible = "fsl,imx6q-gpu";
+			clocks = <&clks 122>, <&clks 74>, <&clks 27>,
+				 <&clks 121>, <&clks 26>;
+			    clock-names = "gpu3d_clk", "gpu3d_shader_clk", "gpu3d_axi_clk",
+					  "gpu2d_clk", "gpu2d_axi_clk";
+			    interrupts = <0 9 0x04>, <0 10 0x04>;
+			    interrupt-names = "irq-3d", "irq-2d";
+			    reg = <0 0>, <0x00130000 0x4000>, <0x00134000 0x4000>;
+			    reg-names = "phys-addr", "iobase-3d", "iobase-2d";
+		};
+
 		ocram: sram@00900000 {
 			compatible = "mmio-sram";
 			reg = <0x00900000 0x20000>;
